kettle/make_json.py

```diff
@@ -149,7 +149,13 @@ def set_elapsed(self):
 def parse_junit(xml):
     """Generate failed tests as a series of dicts. Ignore skipped tests."""
     # NOTE: this is modified from gubernator/view_build.py
-    tree = ET.fromstring(xml)
+
+    try:
+        tree = ET.fromstring(xml)
+    except ET.ParseError:
+        print("Malformed xml, skipping")
```

Comment:
For followup: do we get logging of which tests / jobs had malformed XML?

Reply:
right before hand we log the bucket but within this method we don't have info if it is malformed. 

I will follow up with a PR to make the logging better 👍